lua/lspconfig/sixtyfps.lua

```diff
@@ -0,0 +1,21 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.sixtyfps = {
+    default_config = {
+        cmd = {'sixtyfps-lsp'},
+        filetypes = {'sixtyfps'},
```

Comment:
Is there a filetype plugin users can install so this is recognized?

Reply:
Not yet, but can be added manually at this moment, 'till language development in active progress

Reply:
https://github.com/RustemB/sixtyfps-vim

Reply:
Can you mention that you need this filetype plugin in the docs?

Reply:
ok